scripts/cleanup_failed_registrations.py

```diff
@@ -40,26 +48,27 @@ def remove_failed_registrations(dry_run=True):
             if not f.registered_from:
                 logging.info('Node {0} had registered_from == None'.format(f._id))
                 continue
-            if not f.archive_job:  # Be extra sure not to delete legacy registrations
+            if not f.archive_job:  # Be extra sure not to deal with legacy registrations
                 continue
-            f.archive_job.status = ARCHIVER_FAILURE
-            f.archive_job.sent = True
-            f.archive_job.save()
-            handle_archive_fail(
-                ARCHIVER_UNCAUGHT_ERROR,
-                f.registered_from,
-                f,
-                f.creator,
-                f.archive_job.target_info()
+
+            # Send an email to the OSF Help Desk
+            mails.send_mail(
+                to_addr=settings.SUPPORT_EMAIL,
+                mail=mails.ARCHIVE_UNCAUGHT_ERROR_DESK,
```

Comment:
We should send a different type of email for these "stuck" registrations. Otherwise, it'll be hard to distinguish these from the ones sent here: https://github.com/CenterForOpenScience/osf.io/blob/4a417cc1490052fd019918b5f6e5a809e36104e1/website/archiver/utils.py#L72-L75 . 

How about a new email that says:

```
User: <FULL NAME> (<USERNAME>) [<USER_ID>]

Registration <TITLE> [<NODE_ID>] is stuck in archiving.

Archive Job: [<ARCHIVE_JOB_ID>]

<JSON REPRESENTATION OF ARCHIVE_JOB (can use archive_job.to_storage()) >

Automatically sent from scripts/cleanup_failed_registrations.py
```